langfuse/src/openai/LangfuseSingleton.ts

```diff
@@ -0,0 +1,12 @@
+import { Langfuse } from "../langfuse";
+
+export class LangfuseSingleton {
```

Comment:
Maybe in the next PR, but i think it would also be great to expose the Langfuse configs to the user.